test-scripts/check.sh

```diff
@@ -254,6 +254,42 @@ test_pack_two_snapshots_works() {
   fi
 }
 
+test_pack_snapshots_from_list_works() {
+  rand_megs 1 > ./foo
+  rand_megs 1 > ./bar
+  sha1_ss1=$(cat ./foo | sha1sum)
+  sha1_ss2=$(cat ./bar | sha1sum)
+
+  # Store the first file in SS-1 and second in SS-2
+  printf "foo\n" | "$elfshaker" --verbose store --files-from - SL-1
+  printf "bar\n" | "$elfshaker" --verbose store --files-from - SL-2
+
+  printf "SL-1\n" | "$elfshaker" --verbose pack --compression-level 1 --snapshots-from - PL-1
+  printf "SL-2\n" | "$elfshaker" --verbose pack --compression-level 1 --snapshots-from - PL-2
```

Comment:
Can we also test with 2+ snapshots (e.g.` 'SL-1\nSL-2\n'`) to exercise the input processing a bit more?